### PR TITLE
Glossary pages batch 1

### PR DESCRIPTION
**Pull request by aarondr77**
# Description

Adds first batch of pages to help users transition from Excel to Python by showing how to recreate Excel functionality in Python:
1. REPLACE
2. SUM
3. LOWER
4. CONCAT
5. HOUR

# Testing

Review language / code + make edits directly

# Documentation

No.

### CODE REVIEW

File: trymito.io/excel-to-python-page-contents/CONCAT.json [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+            {
+                "parameter": "text1",
+                "description": "The first text value you want to concatenate.",
+                "dataType": "string"
+            },
+            {
+                "parameter": "text2",
+                "description": "(Optional) Additional text values to concatenate.",
+                "dataType": "string"
+            }
```
Comment (naterush) [inline trymito.io/excel-to-python-page-contents/CONCAT.json:25]: I think we could use some way to represent the ..., aka a variable number of arguments after this.

File: trymito.io/excel-to-python-page-contents/REPLACE.json [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+                ]
+            },
+            {
+                "title": "Replacing specific values in a column",
+                "shortTitle": "Specific values",
+                "paragraphs": [
+                    "To replace specific values in a column in Excel, you'd use the 'Find & Replace' feature or the SUBSTITUTE formula instead of the REPLACE functionality.",
+                    "For more extensive documentation on the SUBSTITUTE formula, refer to the SUBSTITUTE page."
+                ],
+                "codeLines": []
```
Comment (naterush) [inline trymito.io/excel-to-python-page-contents/REPLACE.json:106]: This still appears. I see below you try and handle it, but it doesn't appear to be working. I think you only handle in the commonMistakes section

File: trymito.io/excel-to-python-page-contents/SUM.json [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+        "syntaxTable": [
+            {
+                "parameter": "number1",
+                "description": "The first number you want to sum.",
+                "dataType": "number"
+            },
+            {
+                "parameter": "number2",
+                "description": "(Optional) The second number you want to sum. You can input up to 255 numbers.",
+                "dataType": "number"
```
Comment (naterush) [inline trymito.io/excel-to-python-page-contents/SUM.json:24]: Again, looking for the .. somehow

**Review by naterush — CHANGES_REQUESTED** [review]
These all look excellent, and are very helpful resources. Minor cleanup, then LGTM!

File: trymito.io/excel-to-python-page-contents/REPLACE.json (hunk 2 above)
Comment (aarondr77) [reply trymito.io/excel-to-python-page-contents/REPLACE.json:106]: handled in other PR will merge.

**Pull request merged**